base/src/main/java/com/smartdevicelink/proxy/rpc/BodyInformation.java

```diff
@@ -38,6 +38,7 @@
 import com.smartdevicelink.proxy.rpc.enums.IgnitionStatus;
 
 import java.util.Hashtable;
+import java.util.List;
 
 /**
  * The body information including power modes.

```

Comment:
please replace this
```
 * 			<td>SmartDeviceLink 2.0</td>
 ```

with that:
```
 * 			<td>
 * 				@since SmartDeviceLink 2.0.0
 * 				@deprecated  in SmartDeviceLink 7.1.0
 * 			</td>
 ```

 For `driverDoorAjar`, `passengerDoorAjar`, `rearLeftDoorAjar`, `rearRightDoorAjar` to indicate that they are deprecated

Reply:
@bilal-alsharifi I've made corresponding changes, please, re-review